test/test_cases/test_case_setup.sh

```diff
@@ -1,14 +1,31 @@
 #!/bin/bash
 set -euxo pipefail
 
-# TODO: replace the following with `$cli create secret`
-$cli secrets new-dockercfg dockerpullsecret \
-      --docker-server=${DOCKER_REGISTRY_PATH} \
+announce "Creating image pull secret."
+
+if [[ "${PLATFORM}" == "kubernetes" ]]; then
+     $cli delete --ignore-not-found secret dockerpullsecret
+
+     $cli create secret docker-registry dockerpullsecret \
+      --docker-server=$DOCKER_REGISTRY_URL \
       --docker-username=_ \
-      --docker-password=$($cli whoami -t) \
+      --docker-password=_ \
       --docker-email=_
 
-$cli secrets add serviceaccount/default secrets/dockerpullsecret --for=pull
+elif [[ "$PLATFORM" == "openshift" ]]; then
+
+    $cli delete --ignore-not-found secrets dockerpullsecret
+
+    # TODO: replace the following with `$cli create secret`
+    $cli secrets new-dockercfg dockerpullsecret \
+          --docker-server=${DOCKER_REGISTRY_PATH} \
+          --docker-username=_ \
+          --docker-password=$($cli whoami -t) \
+          --docker-email=_
+
+    $cli secrets add serviceaccount/default secrets/dockerpullsecret --for=pull
```

Comment:
don't we need this for k8s?

Reply:
I do not believe so but I cannot speak to why. `kubernetes-conjur-demo` made this openshift specific. You have any ideas as to why?

Reply:
No